src/Backend/Backend.Infrastructure/Services/Products/ProductService.cs

```diff
@@ -52,22 +53,22 @@ public IEnumerable<Product> Get(Guid tenantId)
         public async Task<Product> Add(Product product)
         {
             var context = LoadContext();
-            product.TenantId = context.Tenant.Id;
-            product = product.Create(product, context.UserId);
+            product.TenantId = context.Tenant.Id; // QUESTION: Should we maintain this?

```

Comment:
Yes, this will asure that you can't create an item in another tenant. A hacker can manipulate the object sent but not the session.